examples/server_certificate.tf

```diff
@@ -0,0 +1,60 @@
+provider "oneview" {
+	ov_username = "Administrator"
+	ov_password = "admin123"
+	ov_endpoint = "https://10.50.9.31/"
+	ov_sslverify = false
+	ov_apiversion = 1200
+	ov_ifmatch = "*"
+}
+
+data "oneview_scope" "scope_obj" {
+        name = "test_scope"
+}
+
+resource "oneview_server_certificate" "ServerCertificate" {
+                                  
+
+
+                                   certificate_details = [{
```

Comment:
In the documentation, you have mentioned that name is mandatory but in examples, the attribute name is not given. Please check the REST API documentation and make it proper.